Packages/GitHub/Sources/GitHubService/GitHubServiceVersion.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public enum GitHubServiceVersion {
+  case dotCom
+  case enterprise(URL)
```

Comment:
These enum cases don't seem to be used. Can we remove them and simplify this enum?